code/modules/clothing/glasses/glasses.dm

```diff
@@ -7,14 +7,14 @@
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
 
 /obj/item/clothing/glasses/science
-	name = "Science Goggles"
+	name = "science goggles"
```

Comment:
Because this PR affects science goggle and it's a minor syntax issue that doesn't warrant its own PR.